README.md

```diff
@@ -96,6 +96,52 @@ piccolo asgi new
 
 [Starlette](https://www.starlette.io/), [FastAPI](https://fastapi.tiangolo.com/), [BlackSheep](https://www.neoteroi.dev/blacksheep/), [Litestar](https://litestar.dev/), [Esmerald](https://esmerald.dev/), [Lilya](https://lilya.dev), [Quart](https://quart.palletsprojects.com/en/latest/), [Falcon](https://falconframework.org/) and [Sanic](https://sanic.dev/en/) are currently supported.
 
+## Piccolo ecosystem
+
+### Piccolo Admin
+
+Piccolo Admin is a powerful admin interface / content management system for Python, built on top of Piccolo.
+
+![Screenshot](https://raw.githubusercontent.com/piccolo-orm/piccolo_admin/master/docs/images/screenshot.png "Screenshot")
```

Comment:
I need to update this image at some point - it's pretty out of date.

Reply:
I agree, but I leave it to you to update the Piccolo Admin readme. I will remove the screenshot from this PR.